rpc-client-api/src/lib.rs

```diff
@@ -1,12 +1,9 @@
 #![allow(clippy::arithmetic_side_effects)]
 
 pub mod client_error;
-pub mod config;
 pub mod custom_error;
```

Comment:
It's too bad, this error type seems like it belongs more in `solana-rpc` since it's really only used there, and isn't actually part of the rpc client